Replicator/tests/ReplicatorSGTest.cc

```diff
@@ -665,6 +665,118 @@ TEST_CASE_METHOD(ReplicatorSGTest, "Pull iTunes deltas from SG", "[.SyncServer][
           timeWithDelta, timeWithoutDelta, timeWithoutDelta/timeWithDelta);
 }
 
+
+TEST_CASE_METHOD(ReplicatorSGTest, "Replicator count balance", "[.SyncServer]") {
```

Comment:
ReplicatorSGTest is never automatically run, since it requires a specially set up SG server. So it's not a good place to put tests that should run in CI. Is there a way you can change this test so it can be moved to ReplicatorLoopbackTest or ReplicatorAPITest?

Reply:
Urhn, I also need SG server to draw out the bug.